x-pack/test/api_integration/apis/security/session.ts

```diff
@@ -9,7 +9,7 @@ import expect from '@kbn/expect/expect.js';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function({ getService }: FtrProviderContext) {
-  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
```

Comment:
__question:__ this is needed because of the change in Kibana behavior that this PR introduces. Although I'd prefer to keep new behavior and consider previous as a bug, I'd like to know what you think: 

Previously if user managed to login somehow and after that their request started to include both cookie **and** `Authorization` header (e.g. user explicitly went to a login page of Kibana that was hosted behind the "auth" proxy and logged in there) we were still prolonging this session for every non-system request even if it was never used to authenticate user (header always took a precedence).

This test did a non-system POST to `/internal/security/session` with both cookie and `Authorization` header (set by `supertest` config) and expected a prolonged cookie in the response, but didn't get anything since request was served by `http`, not `basic` auth header and hence Authenticator didn't automatically extended session lifetime that belonged to another, `basic` provider.